src/ReactNativeSVG.web.ts

```diff
@@ -227,7 +227,7 @@ function remeasure() {
 
```

Comment:
NOTE: all application code changes are as a result of `yarn run lint --fix` and not manually performed